packages/ketcher-react/src/script/ui/views/components/StructEditor/StructEditor.jsx

```diff
@@ -112,12 +112,17 @@ class StructEditor extends Component {
     })
 
     this.editor.event.cursor.add((csr) => {
+      let clientX, clientY
```

Comment:
Is there a specific point for doing this?

Reply:
I would say this may not be the best solution, but it's somehow a not bad workaround. 🤔 

The problem is: Vite seems to have bugs about hoisting decelerations in case block.
![Untitled-2023-04-20-1601](https://user-images.githubusercontent.com/27288153/233301376-e5e00147-a420-4637-a0ae-2a7d494484d5.png)

 😵‍💫It doesn't occur when I set debuggers. It seems that Vite has enough time to decelare(hoist) `clientX` when there is a debugger.

https://user-images.githubusercontent.com/27288153/233313212-c1f8f726-f066-4b61-b55b-d3668a8f57ae.mp4

Why i said that workaround is not bad is because eslint gave warning _Unexpected lexical decelaration in case block_, as you can see in the snapshot. That workaround somehow avoids Vite bug and fixes eslint warning.